tool/test-all-docker.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
`pushd` and `popd` are not necessarily supported by `bin/sh`. I suspect on your system `bin/sh` just symlinks to bash. I'd just make this `bin/bash` to avoid issues.